packages/legacy/core/App/navigators/RootStack.tsx

```diff
@@ -100,7 +100,7 @@ const RootStack: React.FC = () => {
   useEffect(() => {
     async function handleDeepLink(deepLink: string) {
       // If it's just the general link with no params, set link inactive and do nothing
-      if (deepLink.endsWith('//')) {
+      if (deepLink.search(/oob=|c_i=|d_m=/) < 0) {
```

Comment:
should also check for `_oob=` and `_url=`

Reply:
sounds good, it already checks for _oob= and oob= but I'll add url=